patches/emotes/tv.twitch.android.shared.chat.messagefactory.ChatMessageFactory.smali.patch

```diff
@@ -1,25 +1,25 @@
 diff --git a/smali_classes6/tv/twitch/android/shared/chat/messagefactory/ChatMessageFactory.smali b/smali_classes6/tv/twitch/android/shared/chat/messagefactory/ChatMessageFactory.smali
 --- a/smali_classes6/tv/twitch/android/shared/chat/messagefactory/ChatMessageFactory.smali
 +++ b/smali_classes6/tv/twitch/android/shared/chat/messagefactory/ChatMessageFactory.smali
-@@ -667,6 +667,21 @@
+@@ -1101,6 +1101,21 @@
  
-     invoke-static {v15, v1}, Lkotlin/jvm/internal/Intrinsics;->checkNotNullParameter(Ljava/lang/Object;Ljava/lang/String;)V
+     invoke-static {v14, v0}, Lkotlin/jvm/internal/Intrinsics;->checkNotNullParameter(Ljava/lang/Object;Ljava/lang/String;)V
  
 +    # BTTV
 +    #  Add bttv emotes
 +    #     expects:
-+    #       v15: tv.twitch.android.shared.chat.pub.ChatMessageInterface;
-+    invoke-static {v15}, Lbttv/Tokenizer;->retokenizeLiveChatMessage(Ltv/twitch/android/shared/chat/pub/ChatMessageInterface;)V
++    #       v14: tv.twitch.android.shared.chat.pub.ChatMessageInterface;
++    invoke-static {v14}, Lbttv/Tokenizer;->retokenizeLiveChatMessage(Ltv/twitch/android/shared/chat/pub/ChatMessageInterface;)V
 +
 +    # dont add if contians word on blocklist
-+    invoke-static {v15}, Lbttv/api/Blacklist;->isBlocked(Ltv/twitch/android/shared/chat/pub/ChatMessageInterface;)Z
-+    move-result v1
-+    if-eqz v1, :skipnullreturn
-+    const/4 v1, 0x0
-+    return-object v1
++    invoke-static {v14}, Lbttv/api/Blacklist;->isBlocked(Ltv/twitch/android/shared/chat/pub/ChatMessageInterface;)Z
++    move-result v0
++    if-eqz v0, :skipnullreturn
++    const/4 v0, 0x0
++    return-object v0
 +    :skipnullreturn
 +    # /BTTV
 +
-     const-string v1, "webViewSource"
+     const-string v0, "nameColorResolver"
```

Comment:
this indicates that git might have found a similar, but different section, please double check if the patch is applied in the correct function (look for the "webViewSource" string)

Reply:
Yeah, I checked that. This was a part that actually took me a while to understand. The registers in the BTTV code actually rely on the values from the function before it. Twitch added some new stuff (nameColorResolver) but we still needed to be under the same code, rather than above webViewSource (which basically starts a new section).